.github/workflows/ruby.yml

```diff
@@ -0,0 +1,63 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will download a prebuilt Ruby version, install dependencies and run tests with Rake
+# For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
```

Comment:
Nit: this is pretty generic so I think there's not much value having it, when looking for details it's best to lookup the specific action at hand.

Reply:
This was actually part of the template provided by GitHub. Good catch, will remove!